azure-iot-hub/azure/iot/hub/digital_twin_service_client.py

```diff
@@ -0,0 +1,44 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+
+from .auth import ConnectionStringAuthentication
+
+from .protocol.iot_hub_gateway_service_ap_is20190701_preview import (
+    IotHubGatewayServiceAPIs20190701Preview as pl_client,
+)
+from .protocol.models.digital_twin_interfaces import DigitalTwinInterfaces
+from .protocol.models.digital_twin_interfaces_patch import DigitalTwinInterfacesPatch
+
+DigitalTwin = DigitalTwinInterfaces
+DigitalTwinPatch = DigitalTwinInterfacesPatch
+
+
+class DigitalTwinServiceClient(object):
+    def __init__(self, connection_string):
+        self.auth = ConnectionStringAuthentication(connection_string)
+        self.pl = pl_client(self.auth, "https://" + self.auth["HostName"])
```

Comment:
>pl [](start = 13, length = 2)

This should probably be a more readable name - we try not to use these kind of acronyms.

I'd probably go with `protocol_client` or something like that.